h/accounts/events.py

```diff
@@ -5,6 +5,5 @@ def __init__(self, request, user):
 
 
 class LogoutEvent(object):
-    def __init__(self, request, user):
```

Comment:
Not sure I understand why you've removed this. Surely it's useful to know which user just logged out?